pkgs/development/python-modules/complycube/default.nix

```
@@ -0,0 +1,42 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pyhumps
+, pytest
+, python
+, requests
+, setuptools
+}:
+
+buildPythonPackage rec {
+  pname = "complycube";
+  version = "1.1.6";
+  format = "wheel";
+
+  src = fetchPypi rec {
+    inherit version format;
+    pname = "complycube";
+    dist = python;
+    python = "py3";
+    hash = "sha256-vvhRftkB4GJRS3shpmAPR4hKX4T6vHR6oNl6OQSSXYE=";
+  };
+
+  nativeCheckInputs = [ pytest ];
```

```suggestion
  nativeCheckInputs = [ pytestCheckHook ];
```
this allow us to have a stable interface to disable tests